resources/public/api/conf/5.0/uk_property_cumulative_submission_retrieve.yaml

```diff
@@ -0,0 +1,99 @@
+summary: Retrieve a Cumulative UK Property Period Summary
+description: |
+  This endpoint enables you to retrieve the periodic income and expenses for a UK Property business. A National Insurance number, business ID, and tax year are required. Please note that this endpoint is only available for tax years starting from 2025-26.
+
+  ### Test data
+  Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+  
+  | Header Value (Gov-Test-Scenario)    | Scenario                                                                 |
+  |-------------------------------------|--------------------------------------------------------------------------|
+  | N/A - DEFAULT                       | Simulates success response with UK Property.                             |
+  | NOT_FOUND                           | Simulates the scenario where no data is found.                           |
+  | UK_PROPERTY_CONSOLIDATED            | Simulates the scenario with consolidated expenses.                       |
+  | UK_PROPERTY_FULL_EXPENSES           | Simulates success response with full expenses.                           |
+  | FOREIGN_PROPERTY                    | Simulates the scenario where the type of business is incorrect.          |
+  | STATEFUL                            | Performs a stateful retrieve.                                            |
+
+
+tags:
+  - UK Property Cumulative Submission
+
+parameters:
+  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+  - $ref: './common/pathParameters.yaml#/components/parameters/businessId'
+  - $ref: './common/pathParameters.yaml#/components/parameters/taxYear'
+  - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
+  - $ref: './common/headers.yaml#/components/parameters/authorizationReadScopeHeader'
+  - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
+
+security:
+  - User-Restricted:
+      - read:self-assessment
+
+responses:
+  "200":
+    description: Success
+    headers:
+      X-CorrelationId:
+        $ref: './common/headers.yaml#/components/parameters/correlationId'
+      Deprecation:
+        $ref: './common/headers.yaml#/components/parameters/deprecationHeader'
+      Sunset:
+        $ref: './common/headers.yaml#/components/parameters/sunsetHeader'
+      Link:
+        $ref: './common/headers.yaml#/components/parameters/linkHeader'
+    content:
+      application/json:
+        schema:
+          oneOf:
+            - $ref: './schemas/uk_property_cumulative_submission_retrieve/def1/response.json'
+        examples:
+          full-response-for-TY-2025-26:
```

Comment:
I think you can replace this with the kebab-cased name with the english description "Full response for TY 2025-26 and after", then remove the description altogeter. Likewise for the consolidated one.